src/core/gltf2exporter/dracocompressor_p.cpp

```diff
@@ -187,12 +187,12 @@ QVector<T> createIndicesVectorFromAttribute(const Qt3DRender::QAttribute &attrib
     if (byteStride == 0)
     {
         for (int i = 0, nbIndices = indicesVector.size(); i < nbIndices; ++i)
-            indicesVector[i] = static_cast<T>(data.data()[i * sizeof(T)]);
```

Comment:
why not do const T *attributeData = static_cast<T *>(data.constData()); and in the loop indicesVector[i] = attributeData[i]; ?